controllers/servicebinding_controller.go

```diff
@@ -60,7 +60,7 @@ func (r *ServiceBindingReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 // SetupWithManager sets up the controller with the Manager.
 func (r *ServiceBindingReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
-		For(&servicebindingv1alpha3.ServiceBinding{}).
-		Watches(&source.Kind{Type: &servicebindingv1alpha3.ClusterWorkloadResourceMapping{}}, handler.Funcs{}).
+		For(&servicebindingv1beta1.ServiceBinding{}).
```

Comment:
Removing the watch for v1alpha3 without providing a conversion webhook will prevent reconciling for v1alpha3, right?

Reply:
the API Server will "convert" the resources for us without a conversion webhook. In practice it updates the apiVersion and passes the rest of the object unmodified. Since the two versions are otherwise equivalent, this is all we need.